pwnlib/libcdb.py

```diff
@@ -291,6 +291,10 @@ def unstrip_libc(filename):
         log.warn_once('Given libc does not have a buildid. Cannot look for debuginfo to unstrip.')
         return False
 
+    if libc.debuginfo:
+        log.debug("libc contains debug symbols")
```

Comment:
Please mention the consequence in this message too.
"Given libc already contains debug information. Skipping unstrip."

Reply:
Ok sure.
Is there any convention as such that is followed here?

Reply:
No, you just don't see where the message came from, so more context is better